lib/hangman_rules.rb

```diff
@@ -0,0 +1,43 @@
+require_relative 'words'
+
+class HangmanRules
+
+  attr_reader :words, :available_letters, :right_guesses
+
+  def initialize(words)
+  @words = words.get_words()
+  @available_letters = ("a".."z").to_a
+  @right_guesses = []
+  end
+
+  def random_word()
+    words.sample()
+  end
+
+  def keep_track_of_guesses(secret_word, guessed_letter)
+    add_if_right_guess(secret_word, guessed_letter)
```

Comment:
It's a bit odd to see a "_if_" in a method name.
What is the _intent_ of function? What do you need to know from the name here?
How is "add_if_right_guess" a step towards _keeping track of guesses_?
Also, the remove(..) function has no _if_ in the name, but does `.delete_if { ...}` 😄

Reply:
Yeah, that needs to be changed